src/widgets/pixbuf_loader.rs

```diff
@@ -43,7 +43,7 @@ impl PixbufLoader {
     ///
     /// # Failures
     /// Returns `None` if the pixbuf loader cannot be created.
-    pub fn new_with_type(image_type: &str, error: &mut ::glib::Error) -> Option<PixbufLoader> {
+    pub fn new_with_type(image_type: &str, error: &mut Error) -> Option<PixbufLoader> {
```

Comment:
Right, I've missed that comment, sorry.
Generally, it should be okay to return `Err` in case `!error.is_null()` and `assert` that the return value is not null otherwise. And the gdk_pixbuf_loader seems to conforms to this model. The assertion is likely to be moved into a single place in the wrapper types refactoring later...